native/src/modules/endpoint/DatabaseConnector.js

```diff
@@ -5,7 +5,7 @@ import {
   CategoryModel,
   CityModel,
   DateModel,
-  EventModel,
+  EventModel, FeaturedImageModel,
```

Comment:
```suggestion
  EventModel, 
  FeaturedImageModel,
```

Reply:
If linting does not complain then both is fine ;)

Reply:
Agreeing with @Taggotty even though we don't have a rule for this :D Will fix in #93